collection-spec/collection-spec.md

```diff
@@ -0,0 +1,113 @@
+* *processor*: A processor is any provider who processed data to a derived product.
+* *host*: The host is the actual provider offering the data on their storage. There should be no more than one host, specified as last element of the list. 
+
+### Link Object
+
+This object describes a relationship with another entity. Data providers are advised to be liberal with links.
+
+| Field Name | Type   | Description                                                  |
+| ---------- | ------ | ------------------------------------------------------------ |
+| href       | string | **REQUIRED.** The actual link in the format of an URL. Relative and absolute links are both allowed. |
+| rel        | string | **REQUIRED.** Relationship between the current document and the linked document. See chapter "Relation types" for more information. |
+| type       | string | Media type of the referenced entity.                          |
+
+Please see the chapter 'relative vs absolute links' in the [Item spec](../item-spec/item-spec.md#relative-vs-absolute-links) for a discussion on that topic. 
+
+#### Relation types
+
+The following types are commonly used as `rel` types in the Link Object of a Collection:
+
+| Type    | Description                                                  |
+| ------- | ------------------------------------------------------------ |
+| self    | **REQUIRED.** *Absolute* URL to the collection. This is required, to represent the location that the file can be found online. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from. |
+| root    | URL to the root STAC [Catalog](../catalog-spec/README.md) or Collection. |
+| parent  | URL to the parent STAC [Catalog](../catalog-spec/README.md) or Collection. |
+| child   | URL to a child STAC [Catalog](../catalog-spec/README.md) or Collection. |
+| item    | URL to a STAC [Item](../item-spec/README.md). All items linked from a collection MUST refer back to its collection with the `collection` relation type. |
+| license | The license URL for the collection SHOULD be specified if the `license` field is set to `proprietary`. If there is no public license URL available, it is RECOMMENDED to supplement the STAC catalog with the license text in a separate file and link to this file. |
+| derived_from | URL to a STAC Collection that was used as input data in the creation of this collection. See the note in [STAC Item](../item-spec/item-spec.md#relation-types) for more info. |
+
+**Note:** The [STAC Catalog specification](../catalog-spec/catalog-spec.md) requires a link to at least one `item` or `child` catalog. This is _not_ a requirement for collections, but _recommended_. In contrast to catalogs, it is **required** that items linked from a Collection MUST refer back to its Collection with the `collection` relation type.
+
+## Extensions
```

Comment:
Why is the EO extension related? Because it uses bands? 

I don't think I love this section here, especially with no explanation as to why these related extensions are important. I'd prefer it either gone, or else have it actually discuss the relevance a bit, and make clear that the extensions are not mature yet - more ideas. Like mention their extension maturity here. If that's not there then it could feel like these are on the same level as the core spec.

Reply:
Removed EO and scientific, that were old references that more applied to the old dataset proposal. Added some word about the Commons extension, but could surely be improved further.